### ISSUE TITLE
more gcc 11 errors

### ISSUE THREAD

**Issue by Randrianasulu** [heroineworshiper/hvirtual#5]
there are some

```
formattools.C:1035:32: error: ordered comparison of pointer with integer zero ('BC_ListBoxItem*' and 'int')                                  
    1035 |         if(get_selection(0, 0) >= 0)                           
 |            ~~~~~~~~~~~~~~~~~~~~^~~~

```

and one in guicast/bcfilebox.C too

I copied different function for bcfilebox from cin-gg but I guess similar fix can be done here

```
int BC_FileBoxRecent::handle_event()                                   
 {                                                                     
 -       if(get_selection(0, 0) >= 0)                                    
+       BC_ListBoxItem *selection = get_selection(0, 0);            
    +       if(selection != 0)                                                   
   {                                                                               filebox->submit_dir(get_selection(0, 0)->get_text());           
}

```

**Comment by Randrianasulu** [comment]
```
ctracking.C:120:73: error: ordered comparison of pointer with integer zero ('int64_t*' {aka 'long long int*'} and 'int')      
120 |                    mwindow->edl->local_session->view_start > 0)
```

**Comment by Randrianasulu** [comment]
fixed in 91aaed7160f22359b1a62aad7a59f1ae58e804a1
